tests/SprykerEcoTest/Zed/Heidelpay/Business/Facade/IsSalesOrderDebitOnRegistrationCompletedTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Heidelpay\Business;
+
+use Generated\Shared\Transfer\HeidelpayTransactionLogTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use SprykerEco\Shared\Heidelpay\HeidelpayConfig;
+
+/**
+ * @group Functional
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Heidelpay
+ * @group Business
+ * @group Facade
+ * @group ExecuteDebitOnRegistrationTest
+ */
+class IsSalesOrderDebitOnRegistrationCompletedTest extends HeidelpayPaymentTest
+{
+    /**
+     * @return void
+     */
+    public function testIsSalesOrderDebitOnRegistrationCompletedReturnsTrueWhenTransactionLogContainsAtLeastOneSuccessfulRecord(): void
```

Comment:
method naming could be skipped since its already present at class naming
it will make method naming shorter
but it's up to you

if you will update naming, then don't forget to check methos below